gulpfile.js

```diff
@@ -0,0 +1,24 @@
+const del = require('del');
+const eslint = require('gulp-eslint');
+const gulp = require('gulp');
+const zip = require('gulp-zip');
+
+gulp.task('build', ['lint', 'clean:dist'], function () {
+  return gulp.src('src/**')
```

Comment:
@pdehaan hmm, still seems busted. But `src/**/*` seems to work for me. Give it a try, let me know if you see the same behavior.